lib/rspec/core/formatters/exception_presenter.rb

```diff
@@ -244,6 +244,10 @@ def formatted_message_and_backtrace(colorizer)
           end
         end
 
+        def encoded_description(description)
+          encoded_string(description) if description
```

Comment:
I think I'd prefer `return if description.nil?` to make it clear what we're doing with this here

Reply:
Yep. Much better. Thanks Jon